tools/jbrowse2/jbrowse2.xml

```diff
@@ -1129,28 +1146,30 @@ JBrowse2-in-Galaxy
 
 JBrowse2-in-Galaxy offers a highly configurable, workflow-compatible Genome viewer.
 
-Compared to JBrowse1-in-Galaxy, there is no support for alternative codons for unusual genomes,
-and detailed track styling is not yet implemented. Please contact us if you are missing features.
 
 Use and local viewing
 =====================
 
+Browser tracks need a coordinate system, based on the reference genome, so that must be chosen before adding groups of tracks.
 
-A JBrowse2 history item can be opened by viewing it (the "eye" icon).
+At present, 10 Galaxy datatypes that can be turned into a track for display - in all cases, the selected reference genome must have been 
```

Comment:
I would not mention "10". Its hard to update this.

Reply:
Agree. Just happy to have them all :)